_posts/2020-02-04-mirror_mirror.markdown

```diff
@@ -27,18 +27,21 @@ Let's dig into what I've been working on.
 We're going to need a boatload of language extensions to make this work.
 
 ```haskell
-{-# LANGUAGE TypeOperators, AllowAmbiguousTypes  #-}
+{-# LANGUAGE AllowAmbiguousTypes  #-}
 {-# LANGUAGE DataKinds            #-}
+{-# LANGUAGE DeriveAnyClass       #-}
```

Comment:
I followed along at home and these extensions were needed to make the code compile..